data/json/monsters/misc.json

```diff
@@ -211,5 +211,34 @@
     "special_attacks": [ [ "DISAPPEAR", 200 ] ],
     "death_function": { "message": "The %s disappears.", "corpse_type": "NO_CORPSE" },
     "flags": [ "FLIES", "NO_BREATHE", "NOT_HALLUCINATION", "HARDTOSHOOT" ]
+  },
+  {
+    "id": "mon_mannequin_decoy",
+    "type": "MONSTER",
+    "name": { "str": "mannequin decoy" },
+    "description": "A human sized mannequin, making noise.",
+    "looks_like": "mon_shia",
+    "default_faction": "factionless",
+    "species": [ "UNKNOWN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 20,
+    "speed": 1,
+    "symbol": "@",
+    "material": [ "wood" ],
+    "aggression": -99,
+    "morale": 100,
+    "special_attacks": [ [ "PARROT", 1 ], [ "DISAPPEAR", 2000 ] ],
+    "death_drops": {
+      "subtype": "collection",
+      "items": [
+        { "item": "mannequin", "prob": 100, "damage": [ 2, 4 ] },
+        { "item": "talking_doll", "prob": 100, "damage": [ 2, 4 ] }
+      ]
+    },
+    "death_function": { "message": "The %s falls over!", "corpse_type": "NO_CORPSE" },
```

Comment:
So the mannequin just evaporate when it dies? It probably should rather break down like a robot

Reply:
It's right above that line, in death drops. When it dies it yields a damaged mannequin and a damaged talking doll.

Reply:
But it's basically a robot, using the robot death would be better, it would drop a "broken decoy mannequin" that could be deconstructed for component. Plus here it will say "The mannequin desapear" on death and that's weird